src/app.js

```diff
@@ -1,14 +1,16 @@
 const express = require('express')
 const helmet = require('helmet')
 
-function app (config) {
+async function app (config) {
   const app = express()
 
   // Apply the 'helmet' middleware to our express application.  Helmet makes
   // sure that our HTTP headers are correctly configure for security and best
   // practices
   app.use(helmet())
 
+  const tasksBackend = config.tasks.backend
```

Comment:
Do you mind updating `index.js` to include a default/placeholder value for this and/or a comment to document what `config.tasks.backend` is used for?

Reply:
I think it would be helpful to even provide an implementation class that continues to return the dummy message as a demonstration of how we can organize different configurations that gather tasks from different sources.